testutil/singlevm/cleanup.sh

```diff
@@ -21,4 +21,7 @@ sudo docker rm -v -f ceph-demo
 sudo rm /etc/ceph/*
 sudo rm /etc/pki/ciao/controller_key.pem /etc/pki/ciao/ciao-image-key.pem
 sudo rm /etc/pki/ciao/controller_cert.pem /etc/pki/ciao/ciao-image-cacert.pem
+sudo rm /var/lib/ciao/images/4e16e743-265a-4bf2-9fd1-57ada0b28904
```

Comment:
Right now there's no point in starting ciao-image in cleanup.sh as it doesn't have any persistent storage.  Attempts to delete the images via ciao-cli would fail as the newly started image service wouldn't know anything about them, i.e., it would think that there are 0 images.

However, we could also create a new teardown.sh, which could be used to cleanup a correctly initialised cluster using ciao-cli, i.e, 

ciao-cli image delete -all
ciao-cli volume delete -all
ciao-cli instance delete -all.

The existing cleanup.sh would be continue to be used to cleanup the cluster in case of error or interruption, using ciao-launcher --hard-reset, ciao-image --hard-reset.  But this would be done is a separate PR.